plugins/woocommerce-admin/client/shipping/shipping-recommendations.tsx

```diff
@@ -103,7 +103,10 @@ const ShippingRecommendations: React.FC = () => {
 		select( PLUGINS_STORE_NAME ).getActivePlugins()
 	);
 
-	if ( activePlugins.includes( 'woocommerce-services' ) ) {
+	if (
+		activePlugins.includes( 'woocommerce-services' ) ||
+		activePlugins.includes( 'woocommerce-shipping' )
+	) {
```

Comment:
Practically the same things [as here](https://github.com/woocommerce/woocommerce/pull/46929/files#r1629347886) - same conditions and idea. The difference is that area on the WC > Settings > Shipping page will sometimes display this banner here, and sometimes the one in that other place. This depends on the `shipping-smart-defaults` feature flag.

Long story short, different code governing a banner that looks almost the same but we need to disable both, hence the code here.

<img width="720" alt="Screenshot 2024-06-06 at 13 42 54" src="https://github.com/woocommerce/woocommerce/assets/1759681/3af1c0dc-c194-4c43-8989-fd7cfc648194">